### DISCUSSION TITLE
Viewing comments in the Communities section

### DISCUSSION THREAD

**Discussion by MuerteRoja** [FreeTubeApp/FreeTube#3978] [General]
Hello, good afternoon, I have noticed for a while that, in the new update that they have uploaded to the application, v0.19.0 Beta, they have enabled a new function, view the Communities section of the other YouTube channels, more, without However, something has been missing, when I want to see the comments made by other users in the community posts I can't do it, I don't have it allowed, I think it would be interesting if in the future they implement a new function, through which in the application of Freetube you can also read the comments of other users

https://github.com/FreeTubeApp/FreeTube/assets/126993006/8b6751b6-f4ab-4055-830a-d56439828614

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi you're right, there is no way to view the comments right now. This is something that is on our to do list and that's why I made an issue for it https://github.com/FreeTubeApp/FreeTube/issues/3253

We cant guarantee that it will be implemented in the next update but this is definitely on our radar!
